pkg/validation/projects.go

```diff
@@ -23,16 +23,22 @@ func ValidateStarterProjects(starterProjects []v1alpha2.StarterProject) error {
 
 		switch len(gitSource.Remotes) {
 		case 0:
-			errString += fmt.Sprintf("\nstarterProject %s should have at least one remote", starterProject.Name)
+			starterProjectErr := fmt.Errorf("\nstarterProject %s should have at least one remote", starterProject.Name)
+			newErr := resolveErrorMessageWithImportArrtibutes(starterProjectErr, starterProject.Attributes)
+			errString += newErr.Error()
```

Comment:
looks like when appending to errString we added a new line `\n` previously, so as to not make it look like one huge sentence. Could you pls look into that and update it accordingly for both starterProjects and projects?

Reply:
The new line `\n` is still in `starterProjectErr`, and we call  `resolveErrorMessageWithImportArrtibutes` to check for & add import information. so the new line still persists